docs/plugins/Description.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Description
+tags:
+  - plugin/transformer
+---
+
+This plugin generates descriptions that are used in various places:
```

Comment:
generally, description is used for <head>. 

I think it did the same across all emitter.